.github/workflows/pypi.yml

```diff
@@ -28,18 +28,18 @@ jobs:
             cd test-sdist
             python -m venv venv-sdist
             venv-sdist/bin/python -m pip install numpy
-            venv-sdist/bin/python -m pip install ../dist/pymc_experimental*.tar.gz
+            venv-sdist/bin/python -m pip install ../dist/pymc_extras*.tar.gz
             echo "Checking import and version number (on release)"
-            venv-sdist/bin/python -c "import pymc_experimental as pmx; assert pmx.__version__ == '${{ github.ref_name }}'[1:] if '${{ github.ref_type }}' == 'tag' else True; print(pmx.__version__)"
+            venv-sdist/bin/python -c "import pymc as pmx; assert pmx.__version__ == '${{ github.ref_name }}'[1:] if '${{ github.ref_type }}' == 'tag' else True; print(pmx.__version__)"
```

Comment:
This is wrong and elsewhere in this file

Reply:
Oops.